deletetweets.py

```diff
@@ -12,7 +12,7 @@
 __version__ = "0.1"
 
 def delete(api, date, r):
-    with open("tweets.csv") as file:
+    with open("tweets.csv", encoding="utf8") as file:
```

Comment:
Shouldn't the value be "utf-8" (including the dash)?

Reply:
it's not bc with python2 universally, unfortunately it gets a bit funny. Also "utf8" and "utf-8" should be synonymous. I've used this to delete over 17,000 tweets